examples/Tutorials/Enhanced Button/JustRideTests/Info.plist

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
If we don't have tests for the tutorial probably should remove the test files & target